agent/lib/broker_controller.js

```diff
@@ -89,6 +100,28 @@ BrokerController.prototype.sync_addresses = function (addresses) {
     return this.serial_sync();
 };
 
+BrokerController.prototype.sync_addressplans = function (addressplans) {
+    var self = this;
+    var addrs = [];
+    addressplans.forEach(function (addressplan) {
+
+        for (var address in self.addresses) {
+            if (self.addresses[address].plan === addressplan.name) {
+                addrs.push(self.addresses[address]);
+            }
+        }
+    });
+        if (addrs.length) {
+            return Promise.all(addrs.map(function (addr) {
+                return self.update_address_setting(addr);
+            })).then(function () {log.info('updated all ')})
```

Comment:
make this logging more specific, or remove it.

Reply:
removed